### PR TITLE
Migrate firewall startup/shutdown job to bicep based deployment.

### PR DESCRIPTION
**Pull request by gortegaMS**
# Type of PR

- [x] Documentation changes
- [x] Code changes
- [ ] Test changes
- [ ] CI-CD changes
- [ ] GitHub Template changes

## PR Checklist

- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [x] My code follows the code style of this project.
- [x] I ran lint checks locally prior to submission.
- [x] Have you checked to ensure there aren't other open Pull Requests for the same update/change?

## Does this introduce a breaking change

- [ ] YES
- [x] NO

## Validation

- [ ] Unit tests updated and ran successfully
- [ ] Update documentation or issue referenced above

## Issues Closed or Referenced

- Closes: https://github.com/retaildevcrews/wcnp/issues/1159
- References: https://github.com/retaildevcrews/wcnp/issues/1151

### CODE REVIEW

File: scripts/ResourceAutomation/automationAccount.bicep [hunk 1]
```diff
@@ -30,6 +33,12 @@ param scheduleStartOfDayDateTime string = '${dateTimeAdd(utcNow(), 'P1D', 'yyyy-
 @description('End of day datetime for schedule')
 param scheduleEndOfDayDateTime string = '${dateTimeAdd(utcNow(), 'P1D', 'yyyy-MM-dd')}T${scheduleEndOfDayTime}-06:00'
 
+@description('Firewall start of day datetime for schedule')
+param firewallScheduleStartOfDayTime string='08:00:00'
```
Comment (alfredoihernandez) [inline scripts/ResourceAutomation/automationAccount.bicep:37]: can we change the timing between firewall and cluster/gateway to be more than 10 minutes? Ideally 30 to 60 minutes. Also please add item to board to investigate how chain these two together such that we don't need separate schedules.

**Review by alfredoihernandez — CHANGES_REQUESTED** [review]

**Comment by alfredoihernandez** [comment]
can we also as part of this PR remove deprecated powershell scripts for clusters, gateway, and firewalls?

**Comment by alfredoihernandez** [comment]
> can we also as part of this PR remove deprecated powershell scripts for clusters, gateway, and firewalls?

this will be done in later PRs

**Review by alfredoihernandez — APPROVED** [review]

**Comment by gortegaMS** [comment]
- Manually deployed the bicep firewall deployment file from this PR
- Manually disabled the current powershell automation firewall runbooks under RG "rb-wcnp-firewall-automation-dev"
> 
image

**Pull request merged**